src/main/msp/msp.c

```diff
@@ -1444,7 +1444,7 @@ case MSP_NAME:
         break;
 
     case MSP_MOTOR_CONFIG:
-        sbufWriteU16(dst, motorConfig()->minthrottle);
+        sbufWriteU16(dst, 0); // was minthrottle until 4.6
```

Comment:
```suggestion
        sbufWriteU16(dst, 0); // was minthrottle until 4.5
```

Reply:
used `was minthrottle until after 4.5`, unlikely we will back-port it.

Reply:
Backporting is impossible, it's not a bugfix and completely changes CLI settings

Reply:
Other comments for similar situations include:
```
// was currentControlRateProfile->tpa_rate
// not required in API 1.44, was rxConfig()->rcInterpolation
// DEPRECATED: gyro_use_32kHz
// was gyroConfig()->gyro_sync_denom - removed in API 1.43
```
The comment is only intended to clarify that the now null value can't be used for anything in future, and to indicate what it was used to, before firmware 4.6.
I fully appreciate that this MSP value can't be back ported.
If you'd like me to change the comment, to something different, let me know.